### ISSUE TITLE
How to execute shell?

### ISSUE THREAD

**Issue by arun0009** [shockone/black-screen#199]
I am unable to execute shell within black screen terminal.

![image](https://cloud.githubusercontent.com/assets/3824444/12182388/d1e9ac0a-b53c-11e5-9438-650cabc2b171.png)

**Comment by shockone** [comment]
Hey, @arun0009. I'm not sure what it means to execute a shell. Do you mean that executable files that aren't in path can't be executed (like the second command on your screenshot)? Perhaps no, I suspect `bash cassandra` should have produced some output. Could you please attach this file and describe what should have happened?

**Comment by arun0009** [comment]
Yes, I haven't included cassandra in my path and on iTerm I am able to run it by referring to the location of the file (e.g from within bin directory i can run it by ./cassandra). On black screen its not able to execute the file? 

Cassandra file you asked for is at : https://github.com/apache/cassandra/tree/trunk/bin

Screen shot of cassandra execution on iTerm:

![image](https://cloud.githubusercontent.com/assets/3824444/12183875/9f03fc0a-b546-11e5-984c-ebb1e56ceab8.png)

**Comment by shockone** [comment]
Thank you, @arun0009, I was able to reproduce it.
